xwiki-platform-core/xwiki-platform-help/xwiki-platform-help-ui/src/main/resources/XWiki/XWikiSyntaxGroups.es.xml

```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="UTF-8"?>
+<?xml version="1.1" encoding="UTF-8"?>
```

Comment:
I don't think this should have changed.

Reply:
Actually the others are in xml 1.1 so it might be good. See: https://github.com/xwikiorgci/xwiki-platform/blob/master/xwiki-platform-core/xwiki-platform-help/xwiki-platform-help-ui/src/main/resources/XWiki/XWikiSyntaxGroups.de.xml. I'll keep it like that.